src/middlewared/middlewared/plugins/jail.py

```diff
@@ -103,7 +103,7 @@ def query(self, filters=None, options=None):
             self.logger.debug('Failed to get list of jails', exc_info=True)
 
         return filter_list(jails, filters, options)
-    query._fiterable = True
+    query._filterable = True
```

Comment:
I think `@filterable` has been fixed since and it can be used in jail plugin. Can you test, please?